airbyte-integrations/connectors/source-redshift/integration_tests/README.md

```diff
@@ -0,0 +1,3 @@
+# Seeding the dataset
+You can find the SQL scripts in this folder if you need to create or fix the SAT dataset.
+For more instructions and information about valid scripts, please check this [doc](https://docs.google.com/document/d/1k5TvxaNhKdr44aJIHWWtLk14Tzd2gbNX-J8YNoTj8u0/edit#heading=h.ls9oiedt9wyy).
```

Comment:
I don't think we should link a Google Doc here, it's not or usual approach to documentation and can break accessibility. 

Moreover I would suggest you add the following to the README:
- Share the requirements in terms of Redshift setup to run the tests on it.
- Explain the role of each SQL script.
- Explain technically how to seed the DB.

I think these instructions will differ for each source, so I'm not in favor of a central document. If there are common instructions that can be useful for all database sources, feel free to edit https://docs.google.com/document/d/1k5TvxaNhKdr44aJIHWWtLk14Tzd2gbNX-J8YNoTj8u0/edit#heading=h.ls9oiedt9wyy

Reply:
That is the point, all instructions are the same. The difference is only in the credentials required for the connection and the SQL syntax. It's common for any SQL source.
@evantahler created the doc and said to put all related instructions there.

P.S. If we have a new demand to improve or rework something, it's better to create a new issue and discuss it at the planning meeting with the whole team. No need to hold the integration tests for the source as a hostage of this discussion.

Reply:
If I'm being asked to be a tie-breaker, in the spirt of "done is better than perfect", I think the google doc is OK for now.  We can move the contents into the public repo once we are done with all the Java sources, and we can see how similar/different the instructions will be.  Also, the doc currently links to secrets, which is helpful, and shouldn't be in this repo.